File: core/src/galenscovell/oregontrail/ui/components/ShipPanel.java
package galenscovell.oregontrail.ui.components;

import com.badlogic.gdx.scenes.scene2d.ui.Table;
import galenscovell.oregontrail.util.*;

public class ShipPanel extends Table {
    private final GameStage gameStage;

    public ShipPanel(GameStage gameStage) {
        this.gameStage = gameStage;
        construct();
    }

    private void construct() {
        this.setFillParent(true);

        Table mainTable = new Table();
        mainTable.setBackground(ResourceManager.np_test2);

        this.add(mainTable).width(Constants.EXACT_X).height(Constants.EXACT_Y - (Constants.SECTORSIZE * 2)).padTop(Constants.SECTORSIZE * 2);
    }
}

File: core/src/galenscovell/hinterstar/ui/components/ShipPanel.scala
package galenscovell.hinterstar.ui.components

import com.badlogic.gdx.scenes.scene2d.ui.Table
import galenscovell.hinterstar.util._


class ShipPanel(stage: GameStage) extends Table {
  private final val gameStage: GameStage = stage
  construct()


  private def construct(): Unit = {
    this.setFillParent(true)
    val mainTable: Table = new Table
    mainTable.setBackground(ResourceManager.np_test2)
    this.add(mainTable).width(Constants.EXACT_X).height(Constants.EXACT_Y - (Constants.SECTORSIZE * 2)).padTop(Constants.SECTORSIZE * 2)
  }
}
